mocks/tps-mock/src/main/java/no/nav/tjeneste/virksomhet/person/v3/PersonAdapter.java

```diff
@@ -53,6 +54,12 @@ public Bruker fra(PersonModell person) {
         fodselsdato.setFoedselsdato(xcal);
         bruker.setFoedselsdato(fodselsdato);
 
+        // Dødsdato
+        Doedsdato doedsdato = new Doedsdato();
```

Comment:
Blir det riktig å opprette dette objektet i de tilfeller der person.getDødsdato() gir null?

Reply:
Metoden håndtere null, men det er nå unødvendig å opprette objektet når det bare skal settes til null. Legger inn en if(not null).